cmd/notary/integration_nonpkcs11_test.go

```diff
@@ -8,27 +8,18 @@ import (
 	"github.com/docker/notary/passphrase"
 )
 
-func rootOnHardware() bool {
-	return false
-}
-
-// Per-test set up that returns a cleanup function.  This set up changes the
-// passphrase retriever to always produce a constant passphrase
-func setUp(t *testing.T) func() {
-	oldRetriever := retriever
-
-	var fake = func(k, a string, c bool, n int) (string, bool, error) {
-		return testPassphrase, false, nil
-	}
-
+func init() {
+	fake := passphrase.ConstantRetriever("pass")
 	retriever = fake
```

Comment:
I might be missing something, but why do we need both `retriever` and `fake` here?